packages/@okta/vuepress-theme-prose/components/Dialog.vue

```diff
@@ -0,0 +1,131 @@
+        class="c-dialog__title dont-break-out"
+      >
+        Who are you building for today?
+      </h2>
+      <p
+        id="c-dialog-paragraph"
+        class="c-dialog__paragraph dont-break-out"
+      >
+        Explore the developer resources that best fit your needs.
+      </p>
+      <div class="c-dialog__wrapper">
+        <article class="c-dialog__block">
+          <p class="c-dialog__block-paragraph dont-break-out">
+            Workforce Identity Cloud
+          </p>
+          <h4 class="c-dialog__block-title dont-break-out">
+            My employees, <br> contractors, and partners
+          </h4>
+          <p class="c-dialog__block-text c-dialog__paragraph dont-break-out">
+            Workforce Identity Cloud powers identity for all employees, business partners, and contractors and ensures they have access to what they need, when they need it with the appropriate levels of privilege.
+          </p>
+          <button
+            class="c-dialog__link c-dialog__link_type_primary dont-break-out"
+            type="button"
+          >
+            Go to Workforce Identity Cloud docs
+          </button>
+        </article>
+        <article class="c-dialog__block c-dialog__block_type_secondary">
+          <div class="c-dialog__block-wrapper">
+            <p class="c-dialog__block-paragraph dont-break-out">
+              Customer Identity Cloud
+            </p>
+            <p class="c-dialog__block-wrapper-text dont-break-out">
+              Powered by
+            </p>
+            <img
+              class="c-dialog__block-image"
+              src="/img/dialog/auth.svg"
+              width="70"
+              height="29"
+              alt="Auth0"
+            >
+          </div>
+          <h4 class="c-dialog__block-title dont-break-out">
+            My customers <br> or SaaS applications
+          </h4>
+          <p class="c-dialog__block-text c-dialog__paragraph dont-break-out">
+            Customer Identity Cloud enables app builders, digital marketers, and security teams to give end-users access to everything they need online, in a convenient, secure way.
+          </p>
+          <a
+            class="c-dialog__link dont-break-out"
+            href="https://developer.auth0.com"
+          >
+            Go to Customer Identity Cloud docs <span aria-hidden="true">↗</span>
+          </a>
+        </article>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  export default {
+    name: "HomeDialog",
+    mounted() {
+      const dialog = document.querySelector(".c-dialog");
+      const dialogClose = dialog.querySelector(".c-dialog__close");
+      const dialogOverlay = dialog.querySelector(".c-dialog__overlay");
+      const dialogButton = dialog.querySelector(".c-dialog__link_type_primary");
+
+      dialogClose.addEventListener("click", this.hideDialog);
+      dialogOverlay.addEventListener("click", this.hideDialog);
+      dialogButton.addEventListener("click", this.hideDialog);
```

Comment:
Can we use `v-on:click` or `@click` directives instead of using `document.querySelector` to attach these events?